### ISSUE TITLE
Train set equals test set?

### ISSUE THREAD

**Issue by jeroendecloet** [imperial-qore/TranAD#64]
For several datasets (NAB, MBA, SMD & MSDS), the test set is equal to the train set. Why is that?
I thought it was a common ML practice to have the test set completely separate from the training/validation set.

**Comment by shreshthtuli** [comment]
This is just for ease of use and demonstrate how tranad works. The training process does not use any ground-truth labels. Testing compares inferred labels with gold labels.
